Sound/Tidal/Stream.hs

```diff
@@ -218,6 +218,17 @@ infixl 1 |/|
 (|/|) :: ParamPattern -> ParamPattern -> ParamPattern
 (|/|) = mergeNumWith (div) (/)
 
+{- | These are shorthand for merging lists of patterns with @#@, @|*|@, @|+|@,
```

Comment:
just guessing but maybe one of: `##`, `//`, `++` gotcha may already be used in standard haskell